pkgs/tools/package-management/lix/common.nix

```diff
@@ -79,6 +80,25 @@ assert lib.assertMsg (docCargoHash != null || docCargoLock != null)
   "Either `lix-doc`'s cargoHash using `docCargoHash` or `lix-doc`'s `cargoLock.lockFile` using `docCargoLock` must be set!";
 let
   isLegacyParser = lib.versionOlder version "2.91";
+
+  editline-patched = editline.overrideAttrs (prev: {
+    patches = (prev.patches or [ ]) ++ [
+      # Recognize `Alt-Left` and `Alt-Right` for navigating by words in more
+      # terminals/shells/platforms.
+      #
+      # See: https://github.com/troglobit/editline/pull/70
+      # See: https://gerrit.lix.systems/c/lix/+/1883
+      (fetchpatch {
+        url = "https://github.com/troglobit/editline/pull/70/commits/d0f2a5bc2300b96b2434c7838184c1dfd6a639f5.diff";
+        hash = "sha256-0bbtYDUlk1wA0kpTtlaNI6KaCjLmAesZjcWBJZ+DpyQ=";
+      })
+
+      (fetchpatch {
+        url = "https://github.com/troglobit/editline/pull/70/commits/4c4455353a0a88bee09d5f27c28f81f747682fed.diff";
+        hash = "sha256-nVezspwVzeB/8zENeKgwPVum0W1MLv4dOW0967WbX5w=";
+      })
```

Comment:
Could you vendor https://github.com/troglobit/editline/pull/70.patch in the repository instead? GitHub can garbage‐collect unmerged PR commits if they are later force‐pushed, which means that code review in upstreams can cause Nixpkgs builds to fail to reproduce in future.

Reply:
Done!